### PR TITLE
Quake technical service

### PR DESCRIPTION
**Pull request by gclitheroe**
Adds
- a quake technical protobuf message - this has more information about how the quake was located.
- an improved library for parsing SC3ML and converting it to other formats.  
- quake technical protobuf web service.

The sc3ml library should get used to replace some of the funcs under the msg pkg.  I will do this later when we switch from SC3ML version 0.7 -> 0.8

### CODE REVIEW

File: sc3ml/sc3ml_integration_test.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+)
+
+// TestUnmarshalIntegration will list and unmarshal all .xml files
+// under /work/seismcompml07-test and make sure basic quake facts
+// are not zero values.  Run using:
+//
+//     go test -tags=integration
+func TestUnmarshalIntegration(t *testing.T) {
+	var err error
+	var ep EventParameters
```
Comment (junghao) [inline sc3ml/sc3ml_integration_test.go]: This should be eventParameters

File: sc3ml/sc3ml_integration_test.go [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+				t.Fatal(err)
+			}
+			defer f.Close()
+
+			if b, err = ioutil.ReadAll(f); err != nil {
+				t.Fatal(err)
+			}
+			f.Close()
+
+			if ep, err = Unmarshal(b); err != nil {
```
Comment (junghao) [inline sc3ml/sc3ml_integration_test.go]: Should be unmarshal(b)

**Comment by junghao** [comment]
Everything looks good except the 2 typos above.

**Comment by gclitheroe** [comment]
thanks - fixing now.

**Pull request merged**